app/code/Magento/Sitemap/Model/ItemProvider/Composite.php

```diff
@@ -32,7 +32,7 @@ public function getItems($storeId)
     {
         $items = [];
 
-        foreach ($this->itemResolvers as $resolver) {
+        foreach ($this->itemProviders as $resolver) {
             foreach ($resolver->getItems($storeId) as $item) {
                 $items[] = $item;
```

Comment:
Do we really collect all items in array? Why no use generators for lower memory usage?